aws/cf_test.go

```diff
@@ -58,6 +58,32 @@ func TestCreatingStack(t *testing.T) {
 			"fake-stack-id",
 			false,
 		},
+		{
+			"stack with ALB http port",
+			stackSpec{
+				name:             "foo",
+				securityGroupID:  "bar",
+				vpcID:            "baz",
+				loadbalancerType: LoadBalancerTypeApplication,
+				httpTargetPort:   7777,
+			},
+			cfMockOutputs{createStack: R(mockCSOutput("fake-stack-id"), nil)},
+			"fake-stack-id",
```

Comment:
The whole test suite apparently does not test anything besides that mock output matches expected value. 
I've added test to not reduce the coverage but the whole test needs refactoring.

Reply:
I agree